staging/src/k8s.io/apiserver/pkg/server/httplog/httplog.go

```diff
@@ -157,22 +157,23 @@ func (rl *respLogger) Addf(format string, data ...interface{}) {
 func (rl *respLogger) Log() {
 	latency := time.Since(rl.startTime)
 	if klog.V(3).Enabled() {
-		if !rl.hijacked {
-			klog.InfoDepth(1, fmt.Sprintf("verb=%q URI=%q latency=%v resp=%v UserAgent=%q srcIP=%q: %v%v",
-				rl.req.Method, rl.req.RequestURI,
-				latency, rl.status,
-				rl.req.UserAgent(), rl.req.RemoteAddr,
-				rl.statusStack, rl.addedInfo,
-			))
-		} else {
-			klog.InfoDepth(1, fmt.Sprintf("verb=%q URI=%q latency=%v UserAgent=%q srcIP=%q: hijacked",
-				rl.req.Method, rl.req.RequestURI,
-				latency, rl.req.UserAgent(), rl.req.RemoteAddr,
-			))
-		}
+		klog.V(3).InfoS("Recieved HTTP Request", rl.LogArgs(latency)...)
```

Comment:
To replace klog.InfoDepth and keep same source (info in log ouput pointing to code line where it was generated) please move klog.V(3) up the call stack